project.clj

```diff
@@ -34,16 +34,9 @@
                  ;; 'uberjar-exclusions' example toward the end of this file.
                  [org.jruby/jruby-stdlib "1.7.26"]
 
-
-                 [org.clojure/tools.logging "0.3.1"]
```

Comment:
Probably just the latest version available when we created this repo.  Would be good to update it in the parent project but doesn't seem like a blocker.